content/en/docs/reference/kubernetes-api/ref/_index.md

```diff
@@ -0,0 +1,3 @@
+---
+title: "Resources"
+---
```

Comment:
This is a lot of navigation feedback for a three-line space, but here goes 😄 : 

I think we can collapse a lot of unnecessary depth in the reference section as a whole. 

Consider this left nav structure:

![Screen Shot 2020-11-16 at 3 19 58 PM](https://user-images.githubusercontent.com/3210446/99319919-44e6b680-281f-11eb-9365-eb14eefb9752.png)

To clean this up, we need to:

1. Delete the v1.19 page altogether. 

   There's no reason to serve the same content in two forms.

2. Move the "Well-known..." page to the same weight as "API Access Control", immediately following it. 

    It's not part of the API reference _per se_, so let's move it to where it makes better architectural sense.

![Screen Shot 2020-11-16 at 3 23 01 PM](https://user-images.githubusercontent.com/3210446/99320347-4b296280-2820-11eb-9961-83c8c7456144.png)

3. Move Common Definitions out from under Resources.

    It's not a resource definition _per se_, so let's move it to where it makes better architectural sense.

![Screen Shot 2020-11-16 at 3 31 42 PM](https://user-images.githubusercontent.com/3210446/99320770-313c4f80-2821-11eb-874c-08a11b136f92.png)

4. Move Common Parameters out from under Resources and remove the redundant page.

    Like Common Definitions, it's not a resource _per se_, so let's move it.

![Screen Shot 2020-11-16 at 3 34 59 PM](https://user-images.githubusercontent.com/3210446/99320982-a4de5c80-2821-11eb-9b72-34de6e15be8f.png)

   Also, there's no need for a redundant Common Parameters page! Move the nested page up one level.

![Screen Shot 2020-11-16 at 3 38 07 PM](https://user-images.githubusercontent.com/3210446/99321266-4a91cb80-2822-11eb-85bb-e1012cab3c41.png) 

@feloy Please let me know if this makes sense, and if you have any questions.

Reply:
nit: The common definitions should reside with the resources? There is linked content?
nit: I have not studied the output and how it fits with the existing API reference directory, but
I would remove the "Resources" sub-directory and possibly place this content under the versioned directory?

Reply:
Hi @feloy 
Sorry for the trail of feedback. I appreciate the work you have done on behalf of your GSOD project ✨ .
I think the most important piece right now is to make the API reference
navigation/sub-tree (and file hierarchy) as shallow as possible.
What do you think about the suggestion from @zacharysarah?

I'd almost like to see a new directory called `API` off of the root of the left navigation tree (pull out from the Reference section) because there are many more API reference pages. However, a change like this may not be within the bounds of this initial merge. The new API directory could possibly appear after the `Tutorials`.

At a minimum, I would remove the `Resources` directory and place the categories directories directly under the API reference root directory. Also, could move API Reference higher up under the `Reference` sub-tree.

Reply:
Hi @kbhawkey thanks for all your feedbacks, they are very appreciated. 

I agree, and am currently working on the navigation and file structure, based on the remarks by @zacharysarah and you.

I should be able to push a proposition by the end of the week.